protobuf/daemon/servers.proto

```diff
@@ -6,6 +6,11 @@ option go_package = "github.com/NordSecurity/nordvpn-linux/daemon/pb";
 
 import "config/group.proto";
 
+enum ServersError {
+    GET_CONFIG_ERROR = 0;
```

Comment:
Good/common practice is to have one/first item as `no value` or `NO_ERROR` in this case